extras/profile-and-settings.rst

```diff
@@ -86,9 +86,15 @@ Profile Settings
 
    Upload an avatar.
 
-:Password:
+:Password & Auth:
 
-   Change your login password (may be disabled by system admin).
+   Change your login password (may be disabled by system admin) and manage your
+   two-factor authentication methods (if enabled).
```

Comment:
I wonder if it's worth to to expose what is default for the admin to end user here. Maybe both password and 2fa could use same note that they may be disabled by sysadmin? This sounds a bit TMI to me.